src/test/java/com/string/UpperTest.java

```diff
@@ -8,8 +8,8 @@
     @Test
     void testUpper() {
         Assertions.assertEquals(toUpperCase("abc"), ("abc").toUpperCase(), "The strings are equals");
-        //Assertions fail for functional reasons
-        Assertions.assertEquals(toUpperCase("abc"), "abc", "The strings are not equals");
+
+        Assertions.assertNotEquals(toUpperCase("abc"), "abc", "The strings are not equals");
```

Comment:
I think it would be useful if there would be a test something like:

```
Assertions.assertEquals(toUpperCase("abc"), "ABC", "The strings are equal");
```

Just to check if the string is correctly capitalized.